src/sentry/api/endpoints/organization_incident_comment_details.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+from rest_framework import serializers
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.response import Response
+
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.bases.incident import (
+    IncidentEndpoint,
+    IncidentPermission,
+)
+from sentry.api.serializers import serialize
+from sentry.incidents.models import IncidentActivity, IncidentActivityType
+from sentry.incidents.logic import (
+    delete_comment,
+    update_comment,
+)
+
+
+class CommentSerializer(serializers.Serializer):
+    comment = serializers.CharField(required=False)
+
+
+class CommentDetailsEndpoint(IncidentEndpoint):
+    def convert_args(self, request, incident_identifier, activity_id, *args, **kwargs):
```

Comment:
nbd but you can just remove `incident_identifier` and it'll just be in `**kwargs` instead of having to pass it explicitly